api/src/main/java/me/lokka30/treasury/api/common/response/TreasuryException.java

```diff
@@ -0,0 +1,7 @@
+package me.lokka30.treasury.api.common.response;
+
+public class TreasuryException extends Exception{
```

Comment:
Javadoc:
```
The base class for exceptions thrown in Treasury's APIs, such as the {@link me.lokka30.treasury.api.economy.response.EconomyException EconomyException}.

(order of these tags probably isn't right, and unsure if that link will work since it's in a different module)

@author yannicklamprecht
@since {@link me.lokka30.treasury.api.economy.misc.EconomyAPIVersion#v1_0 v1.0}
```